dist/browser.js

```diff
@@ -1827,6 +1827,21 @@ module.exports = class Calculator {
     return bundle;
   }
 
+  /**
+   * To ensure the "Measure Parameter" override is applied to all Libraries.
+   * @param lib
+   */
```

Comment:
Do we need to keep that?

Reply:
Not at all. I commented out the call, but not the function and didn't realize it'd be included in the browser.js.